src/App.tsx

```diff
@@ -1,51 +1,138 @@
 import React from 'react';
+import classNames from 'classnames';
+import { Button } from 'react-bootstrap';
+
 import './App.scss';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+import { Good } from './types/Good';
+
+import goodsFromServer from './goods';
+
+function takeGoodsFromServer(goods: string[]): Good[] {
+  return goods.map((el: string) => ({ name: el }));
```

Comment:
let's add id here

Reply:
Okey.
Added `Id`, but left most of the logic with `name` because it is more obvious how that works in terms of hardcoding initial Jam selection.